src/generator/writeClients.ts

```diff
@@ -89,8 +89,16 @@ function writeClients(dictionary: IReferenceDirectory[]) {
     interfacesTypesImports.forEach((type) => {
       clientFile += `import ${type} from '../interfaces/types/${type}Types';\n`;
     });
-
+    clientFile += `import Transport from '../base/transport'\n`
     clientFile += `\nclass ${fileName}Client {\n`;
+    clientFile += `   
+    private transport: Transport;
+    constructor(transport: Transport) {
```

Comment:
Let's add an empty line here as well, between the `transport` member and the `constructor` :smile:

Reply:
Donne